Sources/NFCPassportReader/PassportReader.swift

```diff
@@ -461,7 +464,7 @@ extension PassportReader {
                 }
             }
             readAttempts += 1
-        } while ( readAttempts < 2 )
+        } while ( readAttempts < 3 )
```

Comment:
I set this to three because of changes made to this function: 
https://github.com/airsidemobile/NFCPassportReader/blob/e99079453fac7a4167e17c4893e1f59819694c4a/Sources/NFCPassportReader/TagReader.swift#L37

```swift
func reduceDataReadingAmount() {
    if useExtendedMode {
        maxDataLengthToRead = 0xFF
        useExtendedMode = false
    } else if maxDataLengthToRead > 0xA0 {
        maxDataLengthToRead = 0xA0
    }
}
```

Basically, it can fail 2 times due to the read size not being supported/compatible.
As an example:
-  First read fails with `useExtendedMode` set to `true`. `maxDataLengthToRead` is then set to `0xFF` and it is retried.
- Second read with `0xFF` fails as-well. `maxDataLengthToRead` set to `0xAO` (to what it is now) and it is retried.
- If the third read fails again, we give up.

Reply:
I see, thanks for the clarification 👍